src/plugins/walletConnectToDapps/scripts/scrapeWalletConnectRegistry.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs')
```

Comment:
can we please move this to the `/scripts` folder such that it's not included into the build? also we may want to convert to typescript and invoke it via a yarn script with `ts-node` - see how we do `yarn release` for reference

Reply:
Additionally, and unless using a package which doesn't support these (which shouldn't be the case, since this is `require`ing the Node.JS standard library and node-fetch both of which should be compatible), please use [ES Modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) and not CommonJS ones - as we conventionally use ESM around the web codebase.